### PR TITLE
Add csv export to good jobs report

### PR DESCRIPTION
**Pull request by iant**

### CODE REVIEW

File: app/views/admin/reports/good_jobs/export.csv.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<%= CSV.generate_line headers %>
+<%- @jobs.each do |job| -%>
+<%= CSV.generate_line(
+  [
+    job['queue_name'],
+    job['job_class'],
+    job['job_id'],
+    job['performed_at'],
+    job['finished_at'],
+    job['time_to_completion_in_seconds']
```
Comment (urbanwide) [inline app/views/admin/reports/good_jobs/export.csv.erb:11]: Assume this is not supposed to be time_to_completion as per the header?

Comment (urbanwide) [reply app/views/admin/reports/good_jobs/export.csv.erb:11]: Otherwise I would suggest using job.values_at(*headers) here. Sorry if I misunderstood

Comment (iant) [reply app/views/admin/reports/good_jobs/export.csv.erb]: :+1:

**Review by urbanwide — APPROVED** [review]
Looks good @iant 👍

**Pull request merged**